joomla/joomla-cms
# Changing hardcoded text to language string in template preview

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/22304

### Summary of Changes

As title says

### Testing Instructions
Allow Preview template in Templates General Options.
Preview template to display the modules positions.

After patching, change the site en-GB.ini new strings values to reflect the fact that `Position:` and `Style:` are no more harcoded

New strings are
JGLOBAL_PREVIEW_POSITION="Position: "
JGLOBAL_PREVIEW_STYLE="Style: "

Verify that you still get:

<img width="1011" alt="screen shot 2018-09-25 at 07 12 32" src="https://user-images.githubusercontent.com/869724/45994118-8392e300-c092-11e8-9eab-585cf71d18e9.png">


Note: No need to test in RTL. I did it. To test it in RTL, it is not enough to set en-GB to rtl. One has to use a RTL string value containing a `:`
